app/settings.py

```diff
@@ -137,3 +137,10 @@ def utcoffset_or_fail(date_value, key):
 ADDRESS_LOOKUP_API_AUTH_TOKEN_LEEWAY_IN_SECONDS = int(
     os.getenv("ADDRESS_LOOKUP_API_AUTH_TOKEN_LEEWAY_IN_SECONDS", "300")
 )
+
+SURVEY_TYPE = os.getenv("SURVEY_TYPE", "census")
+
+
+CENSUS_EN_BASE_URL = "https://census.gov.uk"
+CENSUS_CY_BASE_URL = "https://cyfrifiad.gov.uk"
+CENSUS_NIR_BASE_URL = f"{CENSUS_EN_BASE_URL}/ni"
```

Comment:
I don't think these need to be here anymore and could just be properties in the survey configs?

Reply:
Done